Algorithm/QCAlgorithm.Indicators.cs

```diff
@@ -103,6 +103,30 @@ public AccumulationDistributionOscillator ADOSC(Symbol symbol, int fastPeriod, i
             return accumulationDistributionOscillator;
         }
 
+        /// <summary>
+        /// Creates a new ARIMA indicator.
+        /// </summary>
+        /// <param name="symbol">The symbol whose ARIMA indicator we want</param>
+        /// <param name="p">AR order</param>
+        /// <param name="d">Difference order</param>
+        /// <param name="q">MA order</param>
+        /// <param name="period">Size of the rolling series to fit onto</param>
+        /// <param name="resolution">The resolution</param>
+        /// <returns>The ARIMA indicator for the requested symbol over the specified period</returns>
+        public ArimaIndicator ARIMA(Symbol symbol, int p, int d, int q, int period, Resolution? resolution = null)
```

Comment:
While other names can be used (Autoregressive order (p) Differencing order (d) Moving average order (q)), I initially chose the abbreviated forms for the args and constructor because "[ARIMA(p,d,q)x(P,D,Q)m](https://v8doc.sas.com/sashtml/ets/chap7/sect8.html)" (for the generalized SARIMA, a potential extension class) is usually well understood in time series analysis and thus I figured it could be faster specify/understand for the end user.